core/src/main/scala/filodb.core/query/ResultTypes.scala

```diff
@@ -177,3 +177,9 @@ class QueryLimitException(message: String, queryId: String) extends RuntimeExcep
     s"${super.getMessage}, queryId=${queryId}"
   }
 }
+
+/**
+ * Thrown when number of bytes is greater than container.
+ */
+class RecordOutOfContainerCapacityException() extends RuntimeException(
```

Comment:
Now that we have an exception, it would be nice to know what's the available capacity and what's the requested capacity